packages/orbit-components/src/Table/TableHead/index.tsx

```diff
@@ -1,31 +1,14 @@
-"use client";
-
 import * as React from "react";
-import styled from "styled-components";
 
 import type * as Common from "../../common/types";
-import defaultTheme from "../../defaultTheme";
-import { StyledTableCell } from "../TableCell";
 
-export const StyledTableHead = styled(({ children, className, dataTest }) => (
-  <thead className={className} data-test={dataTest}>
+const TableHead = ({ children, dataTest }: React.PropsWithChildren<Common.Globals>) => (
+  <thead
+    className="border-b-table-head font-table-head w-full whitespace-nowrap border-b"
```

Comment:
I think `font-table-head` should only target table cell children. Although I believe having this style in the parent doesn't hurt, I can't see any side effects. Let me know if it was intentional.

Reply:
I would add `border-solid` explicitly here as well.

Reply:
see my answer above — cascading (in case of `font-table-head` on `thead`) and sensible defaults (`border-solid` is the default) is the most scalable and maintainable way